assert.go

```diff
@@ -79,6 +79,28 @@ func Equal(t testingT, got, want interface{}, opts ...cmp.Option) bool {
 	return assertEqual(t, getArg(1), got, want, opts)
 }
 
+// ErrorContains asserts that the error message contains the wanted string.
+func ErrorContains(t testingT, got error, want string) bool {
+	t.Helper()
+	if got == nil {
+		msg := "got <nil>, want not nil"
+		if expr := getArg(1)(); expr != "" {
+			msg = expr + ": " + msg
+		}
+		t.Error(msg)
+		return false
+	}
+	if !strings.Contains(got.Error(), want) {
+		msg := fmt.Sprintf("got %q, which contain %q", got.Error(), want)
```

Comment:
This message seems inaccurate; is this what you meant?
```suggestion
		msg := fmt.Sprintf("got %q, which does not contain %q", got.Error(), want)
```